keepercommander/commands/helpers/audit_report.py

```diff
@@ -0,0 +1,69 @@
+#  _  __
+# | |/ /___ ___ _ __  ___ _ _ ®
+# | ' </ -_) -_) '_ \/ -_) '_|
+# |_|\_\___\___| .__/\___|_|
+#              |_|
+#
+# Keeper Commander
+# Copyright 2021 Keeper Security Inc.
+# Contact: ops@keepersecurity.com
+#
+from dataclasses import dataclass
```

Comment:
Do we really need to introduce a new dependency?

Reply:
The reason is that I am intentionally trying to break up these very long modules into smaller pieces. I personally feel that most of our files are way too long and need to be broken up. I would be happy to discuss this matter, though. Also if you want me to put all of these back into the main file I will do it.

Reply:
That's fine. Why we need to introduce a new dependency?
Why it cannot be done without adding `dataclasses` dependency?

Reply:
`dataclasses` are just a nice new feature of Python which are standard as of Python 3.7. It makes data objects more readable because the items of the data object have names unlike storing in a tuple or list. Another alternative would be to use a `namedtuple`. `dataclasses` is only needed in  the `install_requires` of the setup.py file for Python < 3.7.

Reply:
I suppose that I am not using that many of the dataclass features and can remove the dependency if you like. Mainly it eliminates the need for the `__init__`.

Reply:
The automated CI testing tests run in Python 3.6 and Python 3.9. However, our tests are not that thorough. It wouldn't be that hard to change the code to work without dataclasses. We should take note, though, that Python 3.6 is only receiving support and security updates through the end of this year. After that, anyone using Python would be advised to upgrade to an officially supported version of Python.

Reply:
I removed the dataclasses dependency